index.html

```diff
@@ -5,9 +5,218 @@
+            <img src="./resources/img/program_icon_02.png" alt="" />
+            <h4>
+              CC <br />
+              Exhibition
+            </h4>
+          </div>
+          <div class="pb__text">
+            <p>
+              Appreciate various creations applying CC license of artists,
+              orginized from Art Center Nabi.
+            </p>
+          </div>
+        </div>
+        <div class="program__boxes--box">
+          .
+          <div class="image-box1">
+            <img src="./resources/img/program_icon_03.png" alt="" />
+            <h4>Forum</h4>
+          </div>
+          <div class="pb__text">
+            <p>
+              Have the time to share your thought and opinions with experts for
+              each topic.
+            </p>
+          </div>
+        </div>
+        <div class="program__boxes--box">
+          <div class="image-box1">
+            <img src="./resources/img/program_icon_04.png" alt="" />
+            <h4>Workshop</h4>
+          </div>
+          <div class="pb__text">
+            <p>
+              Try creating your own work using source license rather than just
+              watching.
+            </p>
+          </div>
+        </div>
+        <div class="program__boxes--box">
+          <div class="image-box1">
+            <img src="./resources/img/program_icon_05.png" alt="" />
+            <h4>CC Ignite</h4>
+          </div>
+          <div class="pb__text">
+            <p>
+              Get opportunities to network with CC affiliates around the world,
+              also after the summit.
+            </p>
+          </div>
+        </div>
+      </div>
+      <div class="program__button btn">
+        <a href="#"> Join CC Alaman Festival 2023</a>
+      </div>
+      <div class="program__link-mobile">
+        <a href="#">See the whole program</a>
+      </div>
+    </section>
+    <section class="featured" id="featured">
+      <div class="featured__heading program__heading">
+        <h3>Featured Artists</h3>
+        <div class="program__hr featured__hr"><hr /></div>
+      </div>
+    </section>
+    <footer>
+      <div class="footer__brands">
+        <div class="featured__heading program__heading partner">
+          <h3>Partner</h3>
+          <div class="program__hr featured__hr"><hr /></div>
+        </div>
+        <div>
+          <ul class="brands__list">
+            <li>mozilla</li>
+            <li>Google</li>
+            <li>NAVER</li>
+            <li>daumkakao</li>
+            <li><i class="fa-brands fa-airbnb"></i>airbnb</li>
+          </ul>
+        </div>
+      </div>
+      <div class="footer__rights">
+        <div class="footer__rights--logo">
+          <h6>
+            Almanac Festival<br />
+            2022
+          </h6>
+        </div>
+        <div class="footer__rights--copyright">
+          2022 Almanac Festival <br />
+          Some rights reserved
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] The footer in index.html on desktop version should be white please try to do that in order to match the design 
![image](https://user-images.githubusercontent.com/61790587/175362308-34808ee0-e529-4df6-904b-73e7e7ecdf81.png)